### PR TITLE
{FEC} Updated existing documents

### PR DESCRIPTION
**Pull request by ethouris**
((DON'T REVIEW BEFORE PARENT IS MERGED: `pf-fec-core-packetfilter-api`))

### CODE REVIEW

**Comment by ethouris** [comment]
BLOCKS: #856

File: docs/API-functions.md [hunk 1]
```diff
@@ -968,7 +968,7 @@ connection parties.
 
 #### SRT_REJ_FILTER
 
-The `SRTO_FILTER` option has been set differently on both connection
+The `SRTO_PACKETFILTER` option has been set differently on both connection
 parties (NOTE: this flag may not exist yet in this version).
```
Comment (stevomatthews) [inline docs/API-functions.md]:
```suggestion
parties (NOTE: this flag does not exist in versions prior to 1.3.x).
```

File: docs/API.md [hunk 2]
```diff
@@ -687,6 +687,17 @@ dropped
 
 ---
 
+| OptName               | Since | Binding | Type   | Units  | Default  | Range   |
+| --------------------- | ----- | ------- | ------ | ------ | -------- | ------- |
+| `SRTO_PACKETFILTER`   | 1.4.0 | pre     | string |        |          | [...512]| 
+
+- **[SET]** - Setup the packet filter. The string must match appropriate syntax
```
Comment (stevomatthews) [inline docs/API.md]:
```suggestion
- **[SET]** - Set up the packet filter. The string must match the appropriate syntax
```

File: srtcore/common.h [hunk 3]
```diff
@@ -589,12 +589,42 @@ class InvertedGuard
 class CSeqNo
 {
 public:
+
+   /// This behaves like seq1 - seq2, in comparison to numbers,
+   /// and with the statement that only the sign of the result matters.
+   /// That is, it returns a negative value if seq1 < seq2,
+   /// positive if seq1 > seq2, and zero if they are equal.
+   /// The only tolerable use of this function is only when you
```
Comment (stevomatthews) [inline srtcore/common.h]:
```suggestion
   /// The only useful application of this function is when you
```

File: srtcore/common.h [hunk 4]
```diff
@@ -589,12 +589,42 @@ class InvertedGuard
...
+   /// compare two values and it works faster than seqoff, however
```
Comment (stevomatthews) [inline srtcore/common.h]:
```suggestion
   /// compare two values and it works faster than seqoff. However
```

File: srtcore/common.h [hunk 5]
```diff
@@ -589,12 +589,42 @@ class InvertedGuard
...
+   /// the result's meaning is only in its sign. DO NOT USE THE
+   /// VALUE for any other purpose and its not meant to be the
```
Comment (stevomatthews) [inline srtcore/common.h]:
```suggestion
   /// VALUE for any other purpose. It is not meant to be the
```

File: srtcore/common.h (hunk 3 above)
Comment (ethouris) [reply srtcore/common.h]: I'd prefer "correct application". The all other uses are not just "useless", but erroneous.

**Pull request merged**